documentation/Trident-API-Overview.md

```diff
@@ -77,13 +77,67 @@ Now suppose you had these tuples with fields ["a", "b", "c"]:
 If you ran this code:
 
 ```java
-mystream.each(new Fields("b", "a"), new MyFilter())
+mystream.filter(new MyFilter())
 ```
 
 The resulting tuples would be:
 
 ```
-[2, 1, 1]
+[1, 2, 3]
```

Comment:
The earlier example had an inputFields param where order of fields were changed ("b", "a") hence the second tuple would match the filter (slightly confusing). In the updated example, since inputFields are not explicitly mentioned the first tuple would match.